force-app/main/default/triggers/ThrowExceptionIfOppStageIsInvalid.trigger

```diff
@@ -0,0 +1,13 @@
+trigger ThrowExceptionIfOppStageIsInvalid on Opportunity (before update) {
```

Comment:
best practice: one trigger per one object, so merge this one with ChangeOnQuoteSentStage trigger.
as we have one trigger per object it should have some common name instead of description of functionality: OpportunityTrigger for this case